src/ep.config.json

```diff
@@ -56,5 +56,9 @@
       "submit_button_url": "",
       "thumbnail_url": ""
     }
+  },
+  "facebook": {
```

Comment:
I'd prefer if this had an enablement toggle similar to our other integrations. Unless are you accepting empty values for pageId & applicationId then will not display the chatbot accordingly?